validation-test/stdlib/FloatingPointConversion.swift.gyb

```diff
@@ -0,0 +1,159 @@
+// RUN: rm -rf %t
+// RUN: mkdir -p %t
+// RUN: %gyb %s -o %t/FloatingPointConversion.swift
+// RUN: %line-directive %t/FloatingPointConversion.swift -- %target-build-swift %t/FloatingPointConversion.swift -Xfrontend -disable-access-control -o %t/a.out_Debug
+// RUN: %line-directive %t/FloatingPointConversion.swift -- %target-build-swift %t/FloatingPointConversion.swift -Xfrontend -disable-access-control -o %t/a.out_Release -O
+//
+// RUN: %line-directive %t/FloatingPointConversion.swift -- %target-run %t/a.out_Debug
+// RUN: %line-directive %t/FloatingPointConversion.swift -- %target-run %t/a.out_Release
+// REQUIRES: executable_test
+
+import StdlibUnittest
+
+
+%{
+
+floatNameToSignificandBits = { 'Float32':24, 'Float64':53, 'Float80':64 }
+
+}%
+
+func getInfiniteOrNaNMessage(_ floatType: String) -> String {
+  if _isDebugAssertConfiguration() {
+    return "either infinite or NaN"
+  }
+  return ""
+}
+
+func getTooSmallMessage() -> String {
+  if _isDebugAssertConfiguration() {
+    return "would be less than"
+  }
+  return ""
+}
+
+func getTooLargeMessage() -> String {
+  if _isDebugAssertConfiguration() {
+    return "would be greater than"
+  }
+  return ""
+}
+
+var FixedPointConversionTraps = TestSuite("FloatingPointToFixedPointConversionTraps")
+var FixedPointConversionFailures = TestSuite("FloatingPointToFixedPointConversionFailures")
+
+var FloatingPointConversionTruncations = TestSuite("FloatingPointToFloatingPointConversionTruncations")
+var FloatingPointConversionFailures = TestSuite("FloatingPointToFloatingPointConversionFailures")
+
+% for Self, selfSignificandBits in floatNameToSignificandBits.iteritems():
+
+%     if Self == 'Float80':
+#if arch(i386) || arch(x86_64)
+%     end
+
+%   for OtherFloat, otherSignificandBits in floatNameToSignificandBits.iteritems():
```

Comment:
Indentation of the second-level `%if` and `%for` is inconsistent, would you mind fixing it?